src/yang/ietf-bgp-types.yang

```diff
@@ -460,10 +470,11 @@ module ietf-bgp-types {
     }
     description
       "union type for route reflector cluster ids:
-      option 1: 4-byte number
-      option 2: IP address";
+       option 1: 4-byte number
+       option 2: IP address";
   }
 
+  // XXX JMH - the community-type name is perhaps improperly named.
```

Comment:
Do you have a suggestion for a better name?

Reply:
Not right now. I'll note that it's already incomplete: it misses large communities.

The motivation for this enumeration tends to be on cisco-like OSes where redistribution of communities is not a default, and you want to restrict what goes where. It's fine for such a feature to exist, but as you add types to it (e.g. long), you combinatorically expand it.

Standard, extended, long, standard+extended, standard+long, extended+long, standard+extended+long, none

Ideally, this would be a bit vector if it's kept.

Reply:
Ok. Made it a bit vector with the following description:

       Type describing variations of community attributes.
       The community types can be combined and a value of 0
       implies 'none'